seed_data/management/commands/authorization_user_exam.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import datetime
+from contextlib import contextmanager
+from factory.django import mute_signals
+from django.core.management import BaseCommand
+
+from courses.models import Program
+from dashboard.api_edx_cache import CachedEdxUserData
+from dashboard.models import CachedEnrollment, ProgramEnrollment
+from dashboard.utils import MMTrack
+from exams.models import ExamAuthorization
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Authorizations of exam to passed users
+    """
+    help = "Trigger exam authorizations when users already passed course"
+
+    def handle(self, *args, **options):
+        """
+        Trigger exam authorizations for user for a course if he has passed it.
+        """
+        programs = Program.objects.filter(exam_series_code__isnull=False, live=True)
+
+        for program in programs:
+            users = ProgramEnrollment.objects.filter(program=program).values_list('user', flat=True)
+
+            for user in users:
+                enrollments_qset = CachedEnrollment.user_course_qset(user, program=program)
+                edx_user_data = CachedEdxUserData(user, program=program)
+                mmtrack = MMTrack(
+                    user,
+                    program,
+                    edx_user_data
+                )
+
+                for enrollment in enrollments_qset:
+                    # if user has passed the course and not already authorized for exam the create authorizations.
+                    ok_for_exam = (
+                        enrollment and
+                        enrollment.course_run and
+                        enrollment.course_run.edx_course_key and
+                        enrollment.course_run.course and
+                        mmtrack.has_passed_course(enrollment.course_run.edx_course_key) and
+                        not ExamAuthorization.objects.filter(user=user, course=enrollment.course_run.course).exists()
+                    )
+                    if ok_for_exam:
+                        ExamAuthorization.objects.create(
+                            user=user,
+                            course=course,
+                            date_first_eligible=datetime.datetime.now() + datetime.timedelta(years=1),
```

Comment:
`date_first_eligible` should be now, `date_last_eligible` can be left as you have it written here.